datadog.go

```diff
@@ -73,6 +74,43 @@ type Options struct {
 	// GlobalTags holds a set of tags that will automatically be applied to all
 	// exported spans.
 	GlobalTags map[string]interface{}
+
+	// DisableCountPerBuckets specifies whether to emit count_per_bucket metrics
+	DisableCountPerBuckets bool
+
+	// EmitPercentiles given a list of percentiles [0.5, 0.95, 0.99], for each one will estimate the percentile
+	// from the Distribution metric and emit a unique metric for each
+	//
+	// Example: []Percentile{{0.5, "50p"},{0.95, "95p"}, {0.99, "99p"}}
+	EmitPercentiles []Percentile
```

Comment:
Thanks a lot for the PR! 

I would like to suggest few things. Mainly for better consistency with the existing histogram implementation:

- https://docs.datadoghq.com/developers/metrics/histograms/
- https://github.com/DataDog/datadog-agent/blob/master/pkg/metrics/histogram.go

More precisely:

1/ I would suggest using a configuration similar to this: https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config_template.yaml#L116-L121

It makes it easier to use and bring more consistency since that avoid mismatch between the percentile and suffix. You can parse/generate both the percentile and suffix based on the list of config `[“0.95”, "0.99”]`.

2/ I would suggest calling it `HistogramPercentiles` instead of `EmitPercentiles` for consistency with what's used in the data-dog-agent histogram implementation: https://github.com/DataDog/datadog-agent/blob/master/pkg/config/config_template.yaml#L116-L121

3/ Result metric suffix `.95percentile` instead of `.95p`.

Reply:
I've changed the options to take in a []float64 which is closer to how the datadog agent configures,but it didn't make sense to take a []string which would have just done a float parse on it.